x-pack/plugins/infra/public/components/asset_details/tabs/metadata/table.tsx

```diff
@@ -168,7 +168,7 @@ export const Table = ({ loading, rows, onSearchChange, search, showActionsColumn
 interface ExpandableContentProps {
   values: string | string[] | undefined;
 }
-const ExpandableContent = (props: ExpandableContentProps) => {
+export const ExpandableContent = (props: ExpandableContentProps) => {
```

Comment:
This is a good candidate to be moved to its own folder within `asset_details`. Seems like a generic component

Reply:
Good idea, I added a new folder for generic components and moved it.